apps/server/src/modules/user/legacy/controller/dto/users-consent.response.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { UserConsentResponse } from './user-consent.response';
+import { ParentConsentResponse } from './parent-consent.response';
+
+export class UsersConsentResponse {
```

Comment:
I think "ConsentsResponse" would be a better name because it holds several consents. The current name sounds like it is for several users.

Reply:
Yeah, but I changes it to UsersConsentResponse since ConsentResponse already exists and there would be confilct in nuxt client

Reply:
I know, but "ConsentsResponse" is not the same as "ConsentResponse", though the difference with the "s" in the middle is admittedly subtle. I still think it is the better name. I definitely do not like "UsersContentResponse" because it is misleading.